src/tests/roc_fec/test_block_writer_reader_errors.cpp

```diff
@@ -384,7 +384,7 @@ TEST(block_writer_reader_errors, reader_cant_decode_packet) {
     // reader should get an error from arena when trying
     // to repair lost packet and shut down
     packet::PacketPtr pp;
-    LONGS_EQUAL(status::StatusAbort, reader.read(pp, packet::ModeFetch));
+    LONGS_EQUAL(status::StatusNoMem, reader.read(pp, packet::ModeFetch));
     CHECK(!pp);
     CHECK(!reader.is_alive());
```

Comment:
Could you also add a check that if we call read() one more time, now it will return StatusAbort?